spec/features/user_contact_index_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe 'Browse contacts', type: :feature do
+  let(:user) { FactoryBot.create :user }
+  let(:contact) { FactoryBot.create :user }
+  let(:social_account) { FactoryBot.create :social_account }
```

Comment:
Why do you need this here?

Reply:
we don't need it anymore
on the other hand: is there a way to get methods from the users helper in rspec?

Reply:
I think you can include a helper like you would in the controller (maybe you need the full path) but I'm not entirely sure.